odkx-src/notify-install.rst

```diff
@@ -0,0 +1,79 @@
+Installing ODK-X Notify
+=======================
+
+.. _notify-prereqs:
+
+Prerequisites
+-------------------
+
+- :doc:`sync-endpoint`
+- :ref:`ODK-X Services<services-using>`
+- :ref:`ODK-X Tables<tables-using>`
+- `Java 8 or higher <https://java.com/en/download/>`_
+
+.. note::
+
+    Users who need to use only Skunkworks-Bat app for receiving notifications, for them only :ref:`ODK-X Services<services-using>` is a prerequisite.
```

Comment:
I am not sure I understand what this note means, which probably means others will not.

Reply:
Removed this note and added separate prerequisites sections for Desktop and Android app for more clarity.